lblod/app-digitaal-loket
# Add LEKP 1.0 (2021) types via extractor

## ID
 DGS-105

 ## Description

When syncing subsidy data from digitaal-loket to app-subsidiedatabank some data in the forms is missing. The reason is because some subjects to export do not have a rdf:type. Without rdf:type we cannot add them to the delta-producer export config file. 

Note: this also removes meta concept schemes that got loaded for nothing as no concept-scheme selectors get used in the forms. 

 ## Type of change


 ## How to test

- Setup the digitaal-loket <--> subsidiedatabank sync. You can follow this guide: https://github.com/lblod/app-subsidiedatabank/wiki/How-to-setup

- Create multiple LEKP 1.0 (2021 !!!) subsidy in digitaal-loket. Fill in the forms differently (act like a user basically trying to break it)
- go over to subsidiedatabank and make sure that all subsidies are a 1 to 1 copy of the ones in loket. (including files)
